fix(styles): dropdown icons in a table are misaligned (#2500)
swisspost/design-system: .changeset/clean-olives-stare.md

---
'@swisspost/design-system-styles': patch
---

> Fixed alignment of icon within button inside datatable.

Reviewer: ```suggestion
Fixed the alignment of icons on datatable buttons.
```
